app/builders/content_change_email_builder.rb

```diff
@@ -8,11 +8,10 @@ def self.call(*args)
   end
 
   def call
-    Email.timed_bulk_insert(
-      columns,
-      records,
-      ProcessAndGenerateEmailsWorker::BATCH_SIZE,
-    )
+    return [] if records.empty?
```

Comment:
Ah, I see you explain the PR comments. Since we've got a couple of repeated pieces of knowledge (timestamps and empty list handling), I'm wondering if we should have some kind of wrapper/helper method to deal with those for us. I realise we're also trying to move away from the custom stuff, so we could also just add comments or ensure the behaviour is covered in tests; as-is, it looks kind of optional.

Reply:
We should be able to remove this conditional soon based on my other PR where we have a guard clause before this: https://github.com/alphagov/email-alert-api/commit/48ecf33e0854a7e448daf28196556507013940bb#diff-7fbb3f7a42ce78544138fb6b12302791R44

Sorry I forgot to add this info on the commit message for this. Will amend

Reply:
Thanks. I notice there are two other occurrences (e.g. [here](https://github.com/alphagov/email-alert-api/pull/1288/files#diff-68c9c8c4027cb03116b9b692da62b20eR13)) that might live for longer, and could lead to an error in production if someone felt the empty array check was unnecessary in future. Worth a test?

Reply:
Oh sure, just added tests for those.